desktop-environment.el

```diff
@@ -182,6 +236,33 @@ portion of the screen."
   (message "New volume value: %s" (desktop-environment-volume-get)))
 
 
+;;; Helper functions - keyboard backlight
+(defun desktop-environment-keyboard-percent ()
```

Comment:
please document every function and variable

Reply:
Done.